docs/Installation.md

```diff
@@ -2,7 +2,14 @@
 
 We suggest to install or use the package in the [Python virtual environment](https://docs.python.org/3/tutorial/venv.html).
 
-If you want to optimize a model from PyTorch, install PyTorch by following [PyTorch installation guide](https://pytorch.org/get-started/locally/#start-locally). For other backend follow: [TensorFlow installation guide](https://www.tensorflow.org/install/), [ONNX installation guide](https://onnxruntime.ai/docs/install/), [OpenVINO installation guide](https://docs.openvino.ai/latest/openvino_docs_install_guides_overview.html).
+NNCF supports multiple backends: OpenVINO&trade;, ONNX, PyTorch, and TensorFlow.
+Ensure that your system meets the specific requirements for the backend you plan to use.
+Below are the details for each backend and links to their installation guides:
+
+- [PyTorch](https://pytorch.org/get-started/locally/#start-locally)
```

Comment:
IMHO: I would suggest listing the backends in one order. 
You wrote `NNCF supports multiple backends: OpenVINO&trade;, ONNX, PyTorch, and TensorFlow.` but in the list PyTorch, Tensorflow, ONNX, OpenVINO